### ISSUE TITLE
Prominent words: Exclude shortcodes

### ISSUE THREAD

**Issue by IreneStr** [Yoast/YoastSEO.js#1076]
We should exclude shortcodes from the prominent words analysis.

This should ensure the following types of 'words' are no longer included:

- review_price
- product_banner (current count = 0, possibly already fixed)
- purchase_link (current count = 0, possibly already fixed)
- yst_add_feed (current count = 0, possibly already fixed)
- product=”site-structure-training (current count = 0, possibly already fixed)

`<under construction>`

**Comment by benvaassen** [comment]
With WP Bakery the following shortcodes are in the Prominent Words section: 

### Steps to reproduce
1.Install WPBakery Page Builder
2.Create a new page, add content using shortcodes
![post-text](https://user-images.githubusercontent.com/1044883/31677020-75c366c8-b337-11e7-866a-161d763ab445.png)

3.See the shortcodes in the Prominent Words section.
![prominent](https://user-images.githubusercontent.com/1044883/31677033-7f2a4d9e-b337-11e7-9a46-1e42bdd8e2a7.png)


### Technical info
* WordPress version:4.8.2
* Yoast SEO version: 5.6.1
* Relevant plugins in case of a bug: WPBakery Page Builder 5.4.2

**Comment by benvaassen** [comment]
Please inform the customer of conversation # 240690 when this conversation has been closed.

**Comment by michaelbriantina** [comment]
Please inform the customer of conversation # 289327 when this conversation has been closed.

**Comment by monbauza** [comment]
Please inform the customer of conversation # 347568 when this conversation has been closed.

**Comment by iamazik** [comment]
Please inform the customer of conversation # 398920 when this conversation has been closed.

**Comment by iamazik** [comment]
The following prominent words appear for the given content in WPBakery Page Builder plugin:
```
1. vc_column_inner width="1/12 offset="vc_col-lg-3 vc_col-md-2 vc_hidden-xs
2. width="1/12 offset="vc_col-lg-3 vc_col-md-2 vc_hidden-xs
3. vc_column_inner width="1/12 offset="vc_col-lg-3 vc_col-md-2
4. offset="vc_col-lg-3 vc_col-md-2 vc_hidden-xs
5. width="1/12 offset="vc_col-lg-3 vc_col-md-2
```
```
[vc_row full_width="stretch_row_content_no_spaces" css=".vc_custom_1528530893149{margin-bottom: 60px !important;}"][vc_column][vc_column_text][metaslider id="1625"][/vc_column_text][/vc_column][/vc_row][vc_row css=".vc_custom_1455123507718{margin-bottom: 0px !important;}"][vc_column][basel_title style="cross" title="FEATURED CATEGORIES" css=".vc_custom_1528527809174{margin-bottom: 20px !important;}" subtitle="MADE WITH LOVE"][vc_row_inner][vc_column_inner width="1/12" offset="vc_col-lg-3 vc_col-md-2 vc_hidden-xs"][/vc_column_inner][vc_column_inner width="5/6" offset="vc_col-lg-6 vc_col-md-8 vc_col-xs-12"][vc_column_text css=".vc_custom_1529721182300{margin-bottom: 25px !important;padding-right: 9% !important;padding-left: 9% !important;}"]
<p class="font-title" style="text-align: center; font-size: 16px; font-style: italic;"><a href="/product-category/unisex-clothing/marvel-tshirt/">A style for every story.</a>
Don’t be ordinary – Be Insane</p>
[/vc_column_text][/vc_column_inner][vc_column_inner width="1/12" offset="vc_col-lg-3 vc_col-md-2 vc_hidden-xs"][/vc_column_inner][/vc_row_inner][basel_categories orderby="title" order="ASC" style="masonry-first" spacing="30" columns="4" title="Categories here" el_class="" number="5" ids="43, 40, 42, 41, 44"][/vc_column][/vc_row][vc_row css=".vc_custom_1455195117811{margin-bottom: 65px !important;}"][vc_column][vc_column_text basel_color_scheme="light" css=".vc_custom_1529721418649{margin-bottom: 0px !important;padding-top: 12px !important;padding-right: 20px !important;padding-bottom: 12px !important;padding-left: 20px !important;background-color: #0a0a0a !important;}"]
<h6 style="text-align: center; line-height: 1.6; letter-spacing: 1.5px; font-size: 14px;">A New Revolution. <a href="/insane-bulk-order/">A New Perspective</a>! · Made With <img class="alignnone" src="/mu-plugins/wpcom-smileys/twemoji/2/svg/2764.svg" alt="theinsane" width="17" height="17" /> By theinsane.</h6>
[/vc_column_text][/vc_column][/vc_row][vc_row css=".vc_custom_1458063964106{margin-bottom: 3.5% !important;}"][vc_column][basel_title style="cross" title="FEATURED PRODUCTS" css=".vc_custom_1528527687722{margin-bottom: 35px !important;}" subtitle="A style for every story."][basel_products items_per_page="8" pagination="arrows" columns="4" order="ASC"][/vc_column][/vc_row][vc_row full_width="stretch_row" css=".vc_custom_1528522952806{margin-bottom: 3% !important;padding-top: 8% !important;padding-right: 25px !important;padding-bottom: 8% !important;padding-left: 25px !important;background-image: url(/wp-content/uploads/2018/06/Join-Our-Newsletter.jpg?id=1605) !important;background-position: center !important;background-repeat: no-repeat !important;background-size: cover !important;}"][vc_column][vc_custom_heading text="Connect to The Insane." font_container="tag:h2|font_size:26|text_align:center|color:%23ffffff|line_height:1" google_fonts="font_family:Lora%3Aregular%2Citalic%2C700%2C700italic|font_style:400%20italic%3A400%3Aitalic"][vc_custom_heading text="Join Our Newsletter" font_container="tag:h2|font_size:58|text_align:center|color:%23ffffff|line_height:1" google_fonts="font_family:Lora%3Aregular%2Citalic%2C700%2C700italic|font_style:400%20regular%3A400%3Anormal"][vc_row_inner][vc_column_inner offset="vc_col-md-offset-4 vc_col-md-4 vc_col-sm-offset-0 vc_col-xs-12"][vc_column_text basel_color_scheme="light" css=".vc_custom_1530677499750{margin-bottom: 25px !important;}"]
<p style="text-align: center;">Sign up it only takes a second to be the first to find out about our latest news and promotions..</p>
[/vc_column_text][/vc_column_inner][/vc_row_inner][vc_column_text basel_color_scheme="light"][mc4wp_form id="173"][/vc_column_text][/vc_column][/vc_row][vc_row content_placement="middle" css=".vc_custom_1485260379029{margin-bottom: 50px !important;}"][vc_column width="1/2"][basel_title style="cross" color="black" title="WHAT THEY SAY ABOUT US"][testimonials slides_per_view="1" hide_prev_next_buttons="yes"][testimonial name="Rahul Raut" title="Happy Customer" image="1622" img_size="100*100"]
<p style="text-align: center;">Superb Quality product. The collection is really nice!
Brought a customized mobile case, fell in love with it.
Want customized stuff? this is the place!</p>
[/testimonial][testimonial name="Sumit Kamble" title="Happy Customer" image="1623" img_size="100*100"]
<p style="text-align: center;">Well crafted website!
Has everything a good website possesses- No overpricing, Voguish collections &amp; unquestionably pretty website!</p>
[/testimonial][/testimonials][/vc_column][vc_column width="1/2"][basel_instagram per_row="3" spacing="1" username="theinsane" number="6"][/basel_instagram][/vc_column][/vc_row][vc_row][vc_column width="1/4"][vc_column_text]
<h2 style="text-align: center;"><strong>THE
OFFICIAL</strong>
<strong>MERCHANDISE</strong>
<strong>STORE</strong></h2>
[/vc_column_text][/vc_column][vc_column width="3/4"][basel_gallery images="1734,1735,1736,1737,1738,1739,1740,1741,1742,1743" view="carousel" spacing="0" slides_per_view="5" hide_pagination_control="yes" hide_prev_next_buttons="yes" autoplay="yes" caption="" img_size="250x210"][/vc_column][/vc_row]
```

## Screenshot
![www theinsane in_insanestaging_wp-admin_post php_post 7 action edit](https://user-images.githubusercontent.com/8946603/42724481-08c34b3c-8795-11e8-90eb-9d4e55cc2e3d.png)


Yoast SEO Version. 7.8
WPBakery Page Builder version: 5.5.2

**Comment by iamazik** [comment]
Please inform the customer of conversation # 410252 when this conversation has been closed.

**Comment by priscillamc** [comment]
Please inform the customer of conversation # 398920  when this conversation has been closed.

**Comment by IreneStr** [comment]
Closing because I cannot reproduce this with the `WPBakery Page Builder` example above in Yoast SEO 14.9, which contains the prominent words overhaul that we released in 14.7.
